deploy/site.yaml

```diff
@@ -23,6 +23,12 @@ env:
       key: secret
       name: launchpad-imagebuild
 
+  - name: CONTRACTS_API_URL
+    configMapKeyRef:
+      key: url
+      name: contracts-api
+      optional: true
```

Comment:
@jpmartinspt I am afraid this property will be ignore, see:
https://github.com/canonical-web-and-design/deployment-configs/blob/master/templates/deployment.yaml#L55-L60